src/app/space-wizard/components/forge-app-generator/forge-app-generator.component.html

```diff
@@ -42,46 +56,37 @@
               <div *ngSwitchCase="'multipleSelection'" style="border:solid 1px #bbb; overflow:hidden;  ; border-radius:4px;">
                 <!--input-->
                 <div class="input-group">
-
                   <div class="input-group-btn">
-                    <div (click)="toggleSelectAll(field)" class="btn btn-default btn-left-multi-select-button">
-
-                      <!--<span [ngClass]="{'glyphicon-check' : field.display.allOptionsSelected,'glyphicon-unchecked' : !field.display.allOptionsSelected}"  class="glyphicon"></span>-->
-                      <span [ngClass]="{'glyphicon-check' : allOptionsSelected(forge.fields[i]),'glyphicon-unchecked' : !allOptionsSelected(forge.fields[i])}"
-                        class="glyphicon"></span>
+                    <div (click)="toggleSelectAll(field)" class="btn btn-default btn-sm btn-left-multi-select-button">
+                      <span [ngClass]="{'fa-check-square-o' : allOptionsSelected(forge.fields[i]),'fa-square-o' : !allOptionsSelected(forge.fields[i])}"
+                        class="fa"></span>
+                    </div>
+                  </div>
+                  <input #optionFilter class="form-control form-control-filter-input" (keyup)="filterList(forge.fields[i],optionFilter.value)"
+                  />
+                  <div class="input-group-btn">
+                    <div class="btn btn-default btn-sm btn-right-multi-select-button">
+                      <span class="fa fa-search"></span>
                     </div>
                   </div>
-
-                  <input class="form-control form-control-filter-input" (keyup)="filterUnselectedList(forge.fields[i],optionFilter.value)"
-                    #optionFilter>
-
                 </div>
                 <!--included list-->
-                <div class="included-list">
-                  <div [class.hidden]="!choice.selected" [class.selected]="choice.selected" *ngFor="let choice of forge.fields[i].display.choices">
-                    <label (click)="deselectOption(forge.fields[i],choice)" class="selectedOption">
-                    <span style="padding-left:5px;padding-right:5px;" class="glyphicon"
-                          [class.glyphicon-check]="choice.selected"
-                          [class.glyphicon-unchecked]="!choice.selected"></span>
-                    {{choice.id}}
-                  </label>
-                    <!--<input [disabled]="fieldOption.selected" style="background-color:transparent; width:3.0rem;margin-left:5px;margin-right:5px; text-align: center; vertical-align: center; display:inline-box; "  type="checkbox" name="{{fieldOption.id}}" [(ngModel)]="fieldOption.selected" class="form-control-" >{{fieldOption.id}}</label>-->
-                  </div>
+                <div style="display:flex;flex-direction:row;flex-wrap:wrap; justify-content:start;" class="included-list">
```

Comment:
@mindreeper2420 please make an issue for this, since it won't get fixed this week.